b.multi_container_pods.md

```diff
@@ -1,7 +1,7 @@
 ![](https://gaforgithub.azurewebsites.net/api?repo=CKAD-exercises/multi_container&empty)
 # Multi-container Pods (10%)
 
-### Create a Pod with two containers, both with image busybox and command "echo hello; sleep 3600". Connect to the second container and run 'ls'
+### Create a Pod with two containers, both with image busybox and command "echo hello; sleep 3600". Connect to the second container and run 'ls'.  View the logs of the first container.
```

Comment:
maybe you mean 'view the logs of the second container'?

Reply:
Yes, my bad